src/main/java/backend/Logic.java

```diff
@@ -279,8 +279,9 @@ public Model getRepo(String repoId) {
     private Optional<TurboIssue> getIssue(String repoId, int issueId) {
         Optional<Model> modelLookUpResult = models.getModelById(repoId);
         return Utility.safeFlatMapOptional(modelLookUpResult,
-                (model) -> model.getIssueById(issueId),
-                () -> logger.error("Model " + repoId + " not found in models"));
+            (model) -> model.getIssueById(issueId),
+            () -> logger.error("Model " + repoId + " not found in models")
```

Comment:
Checkstyle expects line wrapping lambdas to be 4 spaces only and there is no option to tweak that. Refer to https://github.com/checkstyle/checkstyle/issues/281#issuecomment-144733712

The only way to get around might be to ignore checking for that part.